### ISSUE TITLE
[SOLVED] Wyze Cam Pan V3 Support

### ISSUE THREAD

**Issue by raerni** [JoshuaMulliken/ha-wyzeapi#471]
First of all congratulations on your awesome solution to integrate Wyze products into HA with your integration. 

None the less it would be awesome if you could integrate support for the new Wyze cam pan V3, as it currently is not discovered or added in HA according after installing and configuring your HACS integration. I am mainly looking forward to be able to turn the cameras off and on via HA automations or switches. I was able to integrate he video feed without hassle by using the docker wyze bridge.

Thank you and best regards

**Comment by jack5mikemotown** [comment]
Hey, I'm using the wyze cam pan v3 just fine in my home assistant issue. It sounds like the issue may be something on your end

**Comment by Ace0510** [comment]
I have two connected without any issues.

**Comment by JoeSchubert** [comment]
@raerni were your able to figure out what was going on? I assume this is a non-issue since the other two people are able to use those devices fine. I don't have the campan3 to test with. 

If there's no update, I'll assume this is resolved and close it.

**Comment by raerni** [comment]
Hi @jack5mikemotown, @Ace0510 and @JoeSchubert 
My HA does not find them with this Wyze integration. All I get is a "Wyze Notifications" switch. I currently do not use the cloud subscribtion by Wyze... would that be a possible reason? I have not added anything to the configuration.yaml for the Wyze integration. Would I have to? If so, could you point me to the right direction? Maybe I missed something?

Would it be possible that the Wyze cam pan v3 do not get discovered as I am in Switzerland and Wyze does not officially support or sell their products here?

Thanks for trying to help me. I would really love to be able to get more out of the wyze cam pan v3s.

**Comment by raerni** [comment]
Update: I used a different account and stopped the wyze-bridge I was running to get the stream in HA, as I understood it is not possible to get the stream in HA without the wyze-bridge (https://github.com/mrlt8/docker-wyze-bridge). Or can I get the live stream from the cams without the wyze-bridge?

**Comment by raerni** [comment]
SOLUTION:
I stopped my wyze-bridge container and then all my Wyze cam pan v3 cameras got discovered by the Wyze integration. Afterwards I could restart the wyze-bridge, but to get the live feeds via RTSP back I discovered, that the cameras had been commented out in the configuration.yaml (I assume this was done by the wyze integration? as they worked before they had been discovered by the Wyze integration. After uncommenting the camera config block in my configuration.yaml I have the live stream (with some delay because of the RTSP Server) and can control the camera (turn on/off, activate/deactivate motion sensor, turn the siren on/off, as well as notifications.

--> Solved by stopping the wyze-bridge for discovery of the wyze cameras with the wyze integration in HA. Afterwards uncommenting the cameras in configuration.yaml and restarting the wyze-bridge.

Thank you for your help.
best regards
